tests/fixtures/source-code-util/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    // This file is only to ensure no error is thrown due to lack of config.
```

Comment:
I really like that you've included a note explaining why this file exists, but technically comments are not valid in `.json` files (hence the red highlighting from GitHub).  Perhaps it would be better to make this a `.yaml` or `.js` file?